pymc3/variational/svgd.py

```diff
@@ -7,11 +7,13 @@
 import theano
 import theano.tensor as tt
 from tqdm import tqdm
+from .updates import *
```

Comment:
We eradicated `import *` statements from all of the PyMC code base (except the examples). It would be best to import what you need explicitly.